solutions/provisioning_azure_azuredevops/index.asciidoc

```diff
@@ -101,13 +101,8 @@ Adding teams instead of projects is recommended over projects due to https://doc
 * Tracking and auditing: It's easier to link work items and other objects for tracking and auditing purposes
 * Maintainability: You minimize the maintenance of security groups and process updates.
 
-The table below lists typical configurations along with their characteristics:
-[options="header"]
-|=======================
-|Criteria|1 project, N teams      |1 org, N projects/ teams | N orgs
-|General guidance |	Smaller or larger organizations with highly aligned teams | Good when different efforts require different processes (multi) | Legacy migration
-|Process    |Aligned processes across teams; team flexibility to customize boards, dashboards, and so on     |Different processes per prj;e.g. different work item types, custom fields   |same as many projects
-|=======================
+[.internal]
+provisioning_azure_devops_struct
```

Comment:
Internal snippets that are used in solutions currently must be prefixed with `solution_`, because of technical reasons in a synchronization action.